app/views/spree/shared/_upper_footer.html.erb

```diff
@@ -0,0 +1,97 @@
+<div class="footer-links-wrap">
+  <div class="row no-gutters">
+     <div class="col-xl-2 tablet-hide"></div>
+     <div class="col-xl-2 col-lg-4">
+        <div class="footer-logo-wrap">
+           <figure>  <%= image_tag 'footer-logo.svg', alt: "Nature's Flavors" %></figure>
+           <p>Family owned and run for over 40 years. It is our mission to spread the Natural and Organic bounty of nature through our products.</p>
+        </div>
+        <ul class="footer-nav">
```

Comment:
Make the footer links dynamic as per Spree Default.

<%# if spree_menu('footer').present? %>
          <%# spree_menu('footer').children.each do |parent| %>
            <div class="col-3 mb-5">
              <div class="footer-spree-label">
                <%#if parent.container? %>
                  <%#= parent.name %>
                <%# else %>
                  <%#= spree_nav_link_tag(parent) %>
                <%# end %>
              </div>
              <%# parent.children.each do |child| %>
                <div class="pt-2">
                  <%#= spree_nav_link_tag(child) %>
                </div>
              <%# end %>
            </div>
          <%# end %>
        <%# end %>
        
        Please adjust this code inside template to print the links dynamic